### PR TITLE
[hist] THLimitsFinder: survive long long axis ranges.

### PR DESCRIPTION
**Pull request by Axel-Naumann**

### CODE REVIEW

File: tree/dataframe/src/RDFUtils.cxx [hunk 1]
```diff
@@ -322,10 +323,11 @@ void InterpreterDeclare(const std::string &code)
...
 
-Long64_t InterpreterCalc(const std::string &code, const std::string &context)
+Long64_t InterpreterCalc(const std::string &_code, const std::string &context)
 {
    R__LOG_DEBUG(10, RDFLogChannel()) << "Jitting and executing the following code:\n\n" << code << '\n';
 
+   const auto code = "#pragma cling optimize(3)\n" + _code;
    TInterpreter::EErrorCode errorCode(TInterpreter::kNoError);
    auto res = gInterpreter->Calc(code.c_str(), &errorCode);
    if (errorCode != TInterpreter::EErrorCode::kNoError) {
```
Comment (eguiraud) [inline tree/dataframe/src/RDFUtils.cxx:333]: doesn't belong

Comment (Axel-Naumann) [reply tree/dataframe/src/RDFUtils.cxx]: Yeah sorry, fixed!

**Comment by Axel-Naumann** [comment]
@lmoneta test added: I confirmed it fails for master and works with this PR.

**Review by lmoneta — APPROVED** [review]
Thanks Axel, 
PR looks good !

**Comment by Axel-Naumann** [comment]
Failure seems unrelated... Merging.

**Pull request merged**